src/styles/main.css

```diff
@@ -24,21 +27,22 @@ label {
   border-color: #3498DB !important;
 }
 
```

Comment:
These CSS changes make the cards appear non-uniform.  I think ideally each one, with the exception of those without examples, are the same width and the same height.  And they line up from row to row.

![screen shot 2018-06-13 at 10 46 58 am](https://user-images.githubusercontent.com/8315038/41362517-30475236-6ef7-11e8-8469-ff86ac446a82.png)